src/NzbDrone.Core/Datastore/Migration/Framework/MigrationController.cs

```diff
@@ -50,14 +50,15 @@ public void Migrate(string connectionString, MigrationContext migrationContext)
                 {
                     runner.MigrateUp(true);
                 }
+
+                processor.Dispose();
```

Comment:
Possible not called if exception in try that’s not sql, should we move to after?